templates/base.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" integrity="sha384-zCbKRCUGaJDkqS1kPbPd7TveP5iyJE0EjAuZQTgFLD2ylzuqKfdKlfG/eSrtxUkn" crossorigin="anonymous">
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/style.css' %}">
+    {% block title %}
+        <title>Taxi Service</title>
+    {% endblock %}
+</head>
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div id="menu" class="col-sm-3">
+                {% block sidebar %}
+                {% endblock %}
+            </div>
+            <div class="col-sm-6">
+                {% block content %}
+                {% endblock %}
+            </div>
+            <div id="pagination" class="col-sm-3">
+                {% block pagination %}
+                {% endblock %}
+            </div>
+        </div>
+    </div>
+</body>
+</html>
```

Comment:
please, add a blank line at the end of the file

Reply:
trusted Flake8...